src/components/Svgs/PaginationArrow.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { PseudoBox, IconButton } from '@chakra-ui/core';
+import PropTypes from 'prop-types';
+
+/**
+ * @function PaginationArrow
+ *
+ * @param {function} onClick - Callback function that runs when icon is clicked
+ * @param {boolean} isDisabled - Boolean that determines whether the icon is disabled
+ * @param {string} icon - Name of the svg icon from the theme.js file
+ *
+ */
+
+const PaginationArrow = ({ onClick, isDisabled, icon }) => {
+  return (
+    <PseudoBox
+      display="flex"
+      alignItems="center"
+      justifyContent="center"
+      height={10}
+      width={10}
+      stroke="rbb-black"
+      _hover={{
+        color: 'rbb-orange',
+      }}
+      _active={{
+        color: 'white',
+      }}
+      _focus={{
+        color: 'white',
+      }}
+    >
+      <IconButton
+        onClick={onClick}
+        aria-label="Next"
```

Comment:
This should be based on the direction the arrow is pointing. One of the two:

```js
"Go to previous page"
// or
"Go to next page"
```

And it should be applied to the `title` attribute as well for better accessibility for mouse users (hover over something with a `title` attribute and it shows the value to the user).